.github/scripts/test-project.sh

```diff
@@ -119,8 +119,12 @@ elif [ $code -eq 0 ]; then
 
   # confirm existence of correct engine
   echo "-------------- Checking Engines ----------------"
-  bash ../../.github/scripts/check-engines-client.sh $dir $project
-  bash ../../.github/scripts/check-engines-cli.sh $dir $project
+  if [ -z "${SKIP_ENGINE_CHECK+x}" ]; then
```

Comment:
What does `+x` do here?

Reply:
this is just to avoid the unbound variable error because of `set -u`